src/helpers/responses.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Success response
+ * @method
+ * @name successResponse
+ * @param  {String} statusCode status code of the response.
+ * @param  {String} responseCode response code.
+ * @param  {String} message response message.
+ * @param {String} result - result
+ * @returns {JSON} Returns response format
+ */
+const successResponse = ({ statusCode = 200, responseCode = 'OK', message, result = [], meta = {} }) => {
+	let response = {
+		statusCode,
+		responseCode,
+		message,
+		result,
+		meta,
+	}
+
+	console.log('Request Response', { response: response })
```

Comment:
@nevil-mathew ADD degub mode enable/disable in env based on that you can console if debug mode is true

Reply:
@rakeshSgr we do not have any logger service as of now, so we'll have to check in all the places before logging.